modules/logging-release-notes-5-9-10.adoc

```diff
@@ -0,0 +1,24 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-10_{context}"]
+= Logging 5.9.10
+This release includes link:https://access.redhat.com/errata/RHBA-2024:3736[OpenShift Logging Bug Fix Release 5.9.10]
+
+[id="logging-release-notes-5-9-10-bug-fixes"]
+== Bug Fixes
+
+* Before this update, Loki attempted to determine the log level of log messages, which caused confusion when the collector also determined log levels and produced different results. With this update, automatic log level detection in Loki is disabled. (link:https://issues.redhat.com/browse/LOG-6321[LOG-6321])
+
```

Comment:
[LOG-6376](https://issues.redhat.com/browse/LOG-6376) seems to be missing.

Reply:
I've transferred this note to the next PR: https://github.com/openshift/openshift-docs/pull/86355/files#r1888531947.